Code/Source/Wrappers/Texture.cpp

```diff
@@ -0,0 +1,210 @@
+/* Copyright 2020-2021, Robotec.ai sp. z o.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include <Atom/RPI.Public/BlockCompression.h>
+#include <AzCore/Name/NameDictionary.h>
+#include <Wrappers/Texture.h>
+
+namespace RGL::Wrappers
+{
+    Texture Texture::CreateFromMaterialAsset(const AZ::Data::Asset<AZ::RPI::MaterialAsset>& materialAsset)
+    {
+        static const AZStd::string TraceWindow = ConstructTraceWindow(__func__);
```

Comment:
TraceWindow**Name**? Because it is definitely not a window.
+
I would rather use lowercase 't'. Or adjust names below to be consistent.

Reply:
Since this is constant and is the same across different calls, i think the uppercase `T` is appropriate here.